static/js/checks.js

```diff
@@ -36,14 +36,16 @@ $(function () {
         connect: "lower",
         range: {
             'min': [60, 60],
-            '33%': [3600, 3600],
-            '66%': [86400, 86400],
-            '83%': [604800, 604800],
-            'max': 2592000,
+            '25%': [3600, 3600],
+            '45%': [86400, 86400],
+            '55%': [604800, 604800],
+            '70%': [2592000, 2592000],
+            '85%': [5184000, 5184000],
+            'max': 7776000,
```

Comment:
Why is this max higher than the value set in TimeoutForm's `max_value`?

Reply:
I forgot to make the changes in the timeoutForms, let me update that right away